### ISSUE TITLE
Rewrite setup_test.rs in Python

### ISSUE THREAD

**Issue by sydhds** [massalabs/massa#3458]
Following issue #3375, it would be nice to rewrite tools/setup_test.rs (in massa repo) in Python. 

Would be nice to also have this script in massa-sc-runtime too :)

**Comment by Ben-PH** [comment]
Given it's in backlog, I'd be happy to take the issue. It's been awhile since I've played around with python, and would be interested in giving typed python a go.

**Comment by sydhds** [comment]
> Given it's in backlog, I'd be happy to take the issue. It's been awhile since I've played around with python, and would be interested in giving typed python a go.

Let's go and I'll be happy to review it !

**Comment by AurelienFT** [comment]
I think we should wait to we prioritize this refactor.

**Comment by Ben-PH** [comment]
I agree. I'll leave it unclaimed for now, and would be happy to see it taken by someone else that is keen to get it done. 

To clarify, I'm putting my hand up as someone happy to do it :)
